src/components/auth/store/AuthStore.js

```diff
@@ -100,6 +106,21 @@ const errorsSlice = (set, get) => ({
   hasErrors: () => Object.keys(get().errors).length > 0,
 })
 
+export const validationSlice = (set, get) => ({
+  errors: {},
+  hasErrors: () => Object.keys(get().errors).length > 0,
+  getError: (field) => get().errors[field] || [],
```

Comment:
`field` is a little ambigous name, 
better use one of these:
- errorName
- errorKey

do not backup returned value with default, like `[]` in your case. just return `get().errors[field]`